src/app/showcase/colors-showcase/colors-showcase.component.scss

```
@@ -3,62 +3,114 @@
 @each $color, $color-name in $kirby-colors {
```

The variables are reversed:
```suggestion
@each $color-name, $color-value in $kirby-colors {
```